pkg/oc/cli/util/clientcmd/clientcmd.go

```diff
@@ -216,3 +164,27 @@ func (cfg *Config) Clients() (kclientset.Interface, error) {
 
 	return kubeClientset, nil
 }
+
+func DefaultClientConfig(flags *pflag.FlagSet) kclientcmd.ClientConfig {
```

Comment:
is this a duplicate of something else?

Reply:
Yes, this function is also present in `pkg/client/cmd/clientcmd.go`: https://github.com/openshift/origin/pull/17357/files#diff-dc9ab0f48687544bfe9cc38fab5ecac3R207